Documentation/compilation.rst

```diff
@@ -176,7 +209,13 @@ Get your executable with
 
 .. code-block:: bash
 
-   scons -j 32 buildVariablesFile=build/options/supermuc_mac_cluster.py
+   scons -j 4 buildVariablesFile=build/options/supermuc_mac_cluster.py
+
+   # choose a correct executable file in case if you compile multiple
+   # versions of Seissol
+   ln -s $PWD/build/SeisSol_<version> $S3_HOME/bin/Seissol
```

Comment:
This can lead to confusion very quickly, probably better to use the information contained in the exec. name